src/components/navBar/index.tsx

```diff
@@ -0,0 +1,24 @@
+import styles from '@/components/navBar/navBar.module.scss';
+import Image from 'next/image';
+
+const RDSLogo = '/RDSLogo.png';
+
+const NavBar = () => (
+  <div>
+    <nav className={styles.navBar}>
+      <ul>
+        <li>
+          <a style={{ padding: '7px' }} href="https://realdevsquad.com"><Image width="45px" height="45px" src={RDSLogo} alt="real-dev squad" /></a>
```

Comment:
Make a separate class for this too, and move the styles in SCSS file

Reply:
Yeah I tried but I don't know why the styles were not applying using classes or ids.

Reply:
The issue for the css not being applied was that it is being striken out and was not overriding the original white colour of the text which was applied to all list elements.

Solution : Using this -> 

.navBar li .active {}

Instead of this ->

.active {}
![Screenshot (4)](https://user-images.githubusercontent.com/78433013/134054349-016de69d-8c6b-4d04-91c3-b3f4aa4ea8c4.png)